### PR TITLE
HPCC-13370 filter ESP WUQuery by >1 ApplicationName/Value

### PR DESCRIPTION
**Pull request by wangkx**
Also return Application Values for flexible client-side filtering.

Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@afishbeck Please review. Any compatibility concerns?

File: esp/scm/ws_workunits.ecm [hunk 1]
```diff
@@ -347,9 +347,10 @@ ESPrequest [nil_remove] WUQueryRequest
     string LogicalFile;
     string LogicalFileSearchType("");
 
-    string ApplicationName;
-    string ApplicationKey;
-    string ApplicationData;
+    [depr_ver("1.54")] string ApplicationName;
+    [depr_ver("1.54")] string ApplicationKey;
+    [depr_ver("1.54")] string ApplicationData;
```
Comment (afishbeck) [inline esp/scm/ws_workunits.ecm]: I think you want depr_ver("1.55").
I think depr_ver needs the version for which the field becomes deprecated.. in this case depr_ver("1.55").

deper_ver means allow "version < value"
max_ver means allow "version <= value"

**Comment by afishbeck** [comment]
Other than fixing the depr_ver values looks good.  I think the versioning handles backward compatibility.

**Comment by afishbeck** [comment]
@wangkx

**Comment by wangkx** [comment]
Good catch. I will fix it.

**Comment by wangkx** [comment]
@richardkchapman I just fixed the depr_ver values. This PR is ready to be merged.

**Pull request merged**